pkg/minikube/command/kic_runner.go

```diff
@@ -222,8 +221,8 @@ func tempDirectory(isSnap bool) (string, error) {
 	return home, nil
 }
 
-// isSnapBinary returns true if the binary path includes "snap".
-func isSnapBinary() bool {
+// isMinikubeSnap returns true if the minikube binary path includes "snap".
+func isMinikubeSnap() bool {
```

Comment:
consider moving these two funcs on detect package

detect.MinikubeInstalledBySnap()

detect.DockerInstalledBySnap()

Reply:
Didn't know that package existed, thanks for the recommendation, moved